.rubocop_todo.yml

```diff
@@ -58,17 +47,27 @@ RSpec/ContextMethod:
 RSpec/ContextWording:
   Enabled: false
 
-# Offense count: 80
+# Offense count: 85
 # This cop supports unsafe autocorrection (--autocorrect-all).
 # Configuration parameters: SkipBlocks, EnforcedStyle, OnlyStaticConstants.
 # SupportedStyles: described_class, explicit
 RSpec/DescribedClass:
   Enabled: false
 
-# Offense count: 52
+# Offense count: 11
 # This cop supports safe autocorrection (--autocorrect).
 RSpec/EmptyLineAfterFinalLet:
-  Enabled: false
+  Exclude:
```

Comment:
These can be fixed before being added to the todo

Reply:
My plan was to keep this PR confined to the capybara cops but, I can make these changes here if you like: 

https://github.com/DFE-Digital/register-early-career-teachers-public/pull/372/commits/0310985494e6f65574c6fbc0f5fa6af27d9a8dcf